ratatui-widgets/src/block.rs

```diff
@@ -530,6 +534,14 @@ impl<'a> Block<'a> {
         self
     }
 
+    /// TODO documentation
+    /// TODO small example with two block merging / not merging
```

Comment:
I think it doesn't need an example, as it links to MergeStyle, which has examples.
```suggestion
    /// Sets the block's [`MergeStyle`] for overlapping characters. Setting it to `None`
    /// never merges characters.
```